src/index.html

```diff
@@ -3,14 +3,29 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="//db.onlinewebfonts.com/c/c862d8098ce50a7dc042ebbb9b94bbc8?family=TwCenW01-Medium" rel="stylesheet" type="text/css"/> 
     <link rel="stylesheet" href="css/style.scss">
-    <title>Document</title>
+    <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/germano" type="text/css"/> 
+    <title>places-been</title>
 </head>
 <body>
 
-    <picture class='img-container'>
-        <img id ='img' src="" alt="" style="object-fit: cover;">
-    </picture>
+
+    <nav class='navbar '>
+        <h1 class="navbar__title">places been</h1>
+        <h2 class='navbar__subtitle'>a travel journal</h2>
+    </nav>
+
+
+    <main class="gallery container">
+        <div class="buttons container">
+            <h1 class=buttons__view>View:</h1>
+            <button class='button button__visited' type='button'>Visited</button>
+            <button class='button button__buckelist' type='button'>Bucketlist</button>
+            <button class='button button__all' type='button'>Show all</button>
```

Comment:
"View:" does not carry any semantics about heading, so it just ben just a simple span.

One tip about Accessibility. Buttons that serve as a "toggle" (have an active and not active state) should have an `aria-pressed="true"` attribute (true or false).  That way people using a Screen Reader can understand what's the active state. Here's an [article about aria-pressed](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/pressed/)

Hint: To add/remove an attribute with JS you use [`setAttribute`](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)

Reply:
```
    if (activeBtn) {
      activeBtn.classList.remove("active-button");
      current.setAttribute("aria-pressed", false);
    }
```
Why does this work to remove ```.active-button``` but not to set ```aria-pressed='false'```?

Reply:
It must be `activeBtn`, not `current` :)

Reply:
omg I swear I tried that and it didn't work 🤣

Reply:
closing this pr 🎉